terraform/environment/cloudwatch_alarms.tf

```diff
@@ -43,3 +43,37 @@ resource "aws_cloudwatch_metric_alarm" "admin_5xx_errors" {
   threshold                 = 2
   treat_missing_data        = "notBreaching"
 }
+
+resource "aws_cloudwatch_log_metric_filter" "csrf_mistmatch_filter" {
+  name           = "CSRFValuesMismatch"
+  pattern        = "{($.priorityName = \"ERROR\") && ($.message = \"Mismatched CSRF provided\")}"
+  log_group_name = data.aws_cloudwatch_log_group.online-lpa.name
+
+  metric_transformation {
+    name      = "EventCount"
+    namespace = "AWS/ApplicationELB"
```

Comment:
perhaps this metric filter namespace - and the one below is not quite right, as it refers to the load balancer. we probably want a custom namespace for this.